src/pseudopeople/utilities.py

```diff
@@ -1,8 +1,24 @@
+from pathlib import Path
+
 import pandas as pd
+from vivarium.framework.configuration import ConfigTree, ConfigurationError
 from vivarium.framework.randomness import RandomnessStream
 
 from pseudopeople.entities import Form
 
 
 def get_randomness_stream(form: Form, seed: int) -> RandomnessStream:
     return RandomnessStream(form.value, lambda: pd.Timestamp("2020-04-01"), seed)
+
+
+def get_default_configuration() -> ConfigTree:
+    import pseudopeople
+
+    default_config_layers = [
+        "base",
+    ]
+    noising_configuration = ConfigTree(layers=default_config_layers)
+    BASE_DIR = Path(pseudopeople.__file__).resolve().parent
+    yaml_path = BASE_DIR / "default_configuration.yaml"
+    noising_configuration.update(yaml_path, layer="base")
```

Comment:
So this `update` method adds the new keys from the yaml to the base layer? Does it behave just like a dict.update?

Reply:
Yes I believe that is what it is doing according to the docs.  I think the important thing to note is you can update at different levels.

Reply:
Yes; more-or-less. You have the layers, which matter in deciding the "true", desired configuration.